### PR TITLE
set npm config

### PR DESCRIPTION
**Pull request by adriandeka**

### CODE REVIEW

File: api/controllers/ftp-upload/index.js [hunk 1]
```diff
@@ -16,8 +16,8 @@ function uploadFile(config, fullFileName, fileName) {
   const ftpUploadOk = true
   const data = fs.createReadStream(fullFileName)
   const sftp = new sftpClient()
-  const remote = `${config.ftp_test.rootPath}/${fileName}`
-  sftp.connect(config.ftp_test).then(() => {
+  const remote = `${config.ftp.rootPath}/${fileName}`
```
Comment (joel-dekalabs) [inline api/controllers/ftp-upload/index.js:19]: ftp in config file has not rootPath key

**Review by joel-dekalabs — CHANGES_REQUESTED** [review]

**Review by joel-dekalabs — APPROVED** [review]
LGTM

File: api/controllers/ftp-upload/index.js (hunk 1 above)
Comment (adriandeka) [reply api/controllers/ftp-upload/index.js:19]: fixed in [8b93c8a0c2944048902a00abac21deabb763f7ba](https://github.com/XTreamr/aluna_ftp_uploader/pull/2/commits/dd0b659d5dc0147464d54f4de70753f45cd270fc)

**Review by guillaumeparis2000 — APPROVED** [review]
LGTM

**Pull request merged**